Description: Relax dependency on libnfnetlink-1.0.1

Debian wheezy does not carry libnfnetlink-1.0.1, only 1.0.0.

--- ulogd2-2.0.2.orig/configure.ac
+++ ulogd2-2.0.2/configure.ac
@@ -48,7 +48,7 @@ regular_CFLAGS="-Wall -Wextra -Wno-unuse
 AC_SUBST([regular_CFLAGS])
 
 dnl Check for the right nfnetlink version
-PKG_CHECK_MODULES([LIBNFNETLINK], [libnfnetlink >= 1.0.1])
+PKG_CHECK_MODULES([LIBNFNETLINK], [libnfnetlink >= 1.0.0])
 PKG_CHECK_MODULES([LIBMNL], [libmnl >= 1.0.3])
 AC_ARG_ENABLE(nflog,
        AS_HELP_STRING([--enable-nflog], [Enable nflog module [default=yes]]),,[enable_nflog=yes])
